dist/hls.js

```diff
@@ -9963,6 +9992,26 @@ function () {
     var bufferedWithHoles = BufferHelper.bufferInfo(media, currentTime, config.maxBufferHole);
 
     this._tryFixBufferStall(bufferedWithHoles, stalledDuration);
+  } // If current position is outside of buffered range and there is a buffered fragment within `maxFragLookUpTolerance`
+  // will seek to this buffered fragment. It helps to prevent playback stalls
+  ;
```

Comment:
```suggestion
  }; // If current position is outside of buffered range and there is a buffered fragment within `maxFragLookUpTolerance`
  // will seek to this buffered fragment. It helps to prevent playback stalls
```

Reply:
@thomasweld same here